alerts/src/androidLibMain/kotlin/AlertPresenter.kt

```diff
@@ -96,6 +111,43 @@ actual class AlertPresenter(
                 }
             }
             .create()
+            .applyIf(alert.style == Alert.Style.TEXT_INPUT) {
+                val input = EditText(context)
+                alert.textInputAction?.let { textInputAction ->
+                    textInputAction.hint?.let { input.hint = it }
+                    input.inputType = InputType.TYPE_CLASS_TEXT
+                    input.addTextChangedListener(object : TextWatcher {
```

Comment:
please create a helper class to improve readability. the `presentDialog` method is quite long already

Reply:
I created a helper method. A class seemed an overkill. For Android it was more useful as the text watcher is long.